drain_instance.py

```diff
@@ -0,0 +1,43 @@
+import boto3
```

Comment:
This lambda function is written in python because [the original blog post](https://aws.amazon.com/blogs/compute/how-to-automate-container-instance-draining-in-amazon-ecs/) uses python (I completely re-wrote the code though)